.circleci/config.yml

```diff
@@ -3,24 +3,32 @@ workflows:
   version: 2
   tests:
     jobs: &workflow_jobs
-      - node6:
+      - node4:
```

Comment:
Why are we adding node 4 and 9 back to the test config?  Looks like you may need to rebase.